boards/arm/stm32h750b_dk/stm32h750b_dk.dts

```diff
@@ -81,3 +83,42 @@
 	current-speed = <115200>;
 	status = "okay";
 };
+
+&quadspi {
+	pinctrl-names = "default";
+	pinctrl-0 = <&quadspi_clk_pf10 &quadspi_bk1_ncs_pg6
+		     &quadspi_bk1_io0_pd11 &quadspi_bk1_io1_pf9
+		     &quadspi_bk1_io2_pf7 &quadspi_bk1_io3_pf6
+		     &quadspi_bk2_io0_ph2 &quadspi_bk2_io1_ph3
+		     &quadspi_bk2_io2_pg9 &quadspi_bk2_io3_pg14>;
+	flash-id = <1>;
+	status = "okay";
+
+	mt25ql512ab1: qspi-nor-flash-1@0 {
+		compatible = "st,stm32-qspi-nor";
+		reg = <0>;
+		qspi-max-frequency = <72000000>;
+		size = <DT_SIZE_M(512)>; /* 64 MBytes */
+		spi-bus-width = <4>;
+		status = "okay";
+
+		partitions {
+			   compatible = "fixed-partitions";
+			   #address-cells = <1>;
+			   #size-cells = <1>;
+
+			   partition@0 {
+			       reg = <0x0 DT_SIZE_M(64)>;
+			   };
+		};
+	};
+
+	mt25ql512ab2: qspi-nor-flash-2@1 {
+		compatible = "st,stm32-qspi-nor";
+		reg = <1>;
```

Comment:
In other places `reg = 0` is used, but that gave a dtc warning, so I made this 1. Am I correct to assume the exact number doesn't matter in this case?

Reply:
Does not really matter yet. 
Later I'd like to define  this property  as  the external flash base address and size, for example : 
`reg = <0x90000000 DT_SIZE_M(64)>; /* 512 Mbits */`